.github/workflows/build.yaml

```diff
@@ -19,4 +19,6 @@ jobs:
         run: sudo apt install nmap -y
 
       - name: Go test
-        run: go test -v ./... -count 1
+        run: |
+          go test ./internal/http/... -count=1 -race
+          go test -v -count 1 ./...
```

Comment:
Add a TODO comment explaining that, once the known race conditions are fixed, this should be changed to:

```
go test -cover -race -count 1 ./...
```

I'd also add the `-cover` flag and remove `-v`, which adds a lot of noise.

Reply:
Done in79ab287f168b84ec65314e3027bd4c8df86693db